### DISCUSSION TITLE
Vercel Deployment

### DISCUSSION THREAD

**Discussion by rghadamian** [msoedov/langcorn#30] [Q&A]
Hi there

I've been trying to get Python working on my NextJs app with Turborepo on Vercel but I can't seem to get it to work and keep coming up with this error. 
<img width="656" alt="Screenshot 2023-08-21 at 3 25 28 pm" src="https://github.com/msoedov/langcorn/assets/7765692/ffe419fd-b00a-4586-bd97-b4535b2bf2b4">

I'm wondering if you had some suggestions on:

1. how I need to amend the vercel.json
2. where I should put the vercel.json file based on this file structure? in the root file where turbo.json is or in the app root file?

<img width="204" alt="Screenshot 2023-08-21 at 3 27 03 pm" src="https://github.com/msoedov/langcorn/assets/7765692/623697d5-26e9-4752-8e6a-7e3cb1326f68">

I have a apps/api/src/api folder for everything Typescript, and have added a apps/api/api folder for everything Python as suggested for Vercel but not sure if you can mix both together?

**Comment by msoedov** [comment]
I don't know if vercel supports multiple deployments from a mono repo. I have a working example for `vercel.json` config in this repo.
